database/sqlite/sqlite_aclk.h

```diff
@@ -67,7 +67,7 @@ static inline void uuid_unparse_lower_fix(uuid_t *uuid, char *out)
 }
 
 #define TABLE_ACLK_ALERT "CREATE TABLE IF NOT EXISTS aclk_alert_%s (sequence_id INTEGER PRIMARY KEY, " \
-        "alert_unique_id, date_created, date_submitted, date_cloud_ack, " \
+        "alert_unique_id, date_created, date_submitted, date_cloud_ack, filtered_alert_unique_id, " \
```

Comment:
Why are we not adding constraint here (at least a `NOT NULL`)? :thinking:

Reply:
@MrZammler your PR is working as expected, but the missing constraint is something that I still cannot understand. :thinking:

Reply:
Hi Thiago!

If you mean a constrain on `filtered_alert_unique_id`, then, although we should make sure there is not entry in the table without it, in case an insert happens without this field, it should error out on the insert. We need to do the insert for the alert to be queued, it's just the filter won't work.

In any case, I will check the code to make sure the field is always populated.

Reply:
Added the constraint!